src/instructlab/defaults.py

```diff
@@ -26,6 +26,17 @@
 LOG_FORMAT = "%(levelname)s %(asctime)s %(name)s:%(lineno)d: %(message)s"
 
 
+class ILAB_PROCESS_MODES:
+    DETACHED: str = "detached"
+    ATTACHED: str = "attached"
+
+
+class ILAB_PROCESS_TYPES:
+    DATA_GENERATION: str = "Generation"
+
+    TRAINING: str = "Training"
+
```

Comment:
Not sure why there's an extra newline here
```suggestion
class ILAB_PROCESS_TYPES:
    DATA_GENERATION: str = "Generation"
    TRAINING: str = "Training"

```

Reply:
tox adds this in, keeping it

Reply:
How is `tox` doing this?

Reply:
`tox -e fix` adds this in if I remove it

Reply:
who am I to argue with the great and powerful `tox`

btw you can run `make fix` as shorthand (if you aren't already)

resolving this